.github/component_owners.yml

```diff
@@ -7,15 +7,13 @@ components:
     - cijothomas
     - codeblanch
     - rajkumar-rangaraj
-    - reyang
     - utpilla
   src/OpenTelemetry.Exporter.InfluxDB/:
     - havret
   src/OpenTelemetry.Exporter.Instana/:
     - zivaninstana
   src/OpenTelemetry.Exporter.OneCollector/:
     - codeblanch
```

Comment:
It will be great to have second owner in case of @CodeBlanch vacation.

Reply:
@rajkumar-rangaraj Are you willing to co-own `OneCollectorExporter`?

Reply:
I don't think it is a blocker for this PR, agree?